src/index.html

```diff
@@ -1,13 +1,76 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+     rel="preconnect" 
+     href="https://fonts.gstatic.com" 
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" 
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./main.css">
+  </head>
+  <body>
+    <div class="wrapper">
+      <div
+        data-qa="card" 
+        class="card card--position_auto"
+      > 
+        <div class="card__image card__image--dist_bottom">
+          <a href="#"
+            class="card__link">
+            <img 
+              src="images/imac.jpeg" 
+              alt="APPLE A1419 iMac 27 Retina 5K Monoblock " 
+              class="card__link-image"
+            >
+          </a>
+        </div>
+
+        <a 
+          href="#" 
+          class="card__product"
+        >
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
+        <div class="card__code card__code--dist_bottom">
+          Product code: 195434
+        </div>
+
+        <a 
+          href="#" 
+          class="card__stars stars-container stars-container--dist_bottom"
+        >
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="card_review">
```

Comment:
1. Empty line between multiline siblings
2. Correct separator for BEM classes

```suggestion
          </div>
          
          <span class="card__review">
```